.github/workflows/ci.yml

```diff
@@ -113,6 +113,28 @@ jobs:
         python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
         python builder.pyz build -p ${{ env.PACKAGE_NAME }}
 
+  windows-vc16:
+    runs-on: windows-2022 # latest
+    strategy:
+      matrix:
+        arch: [x86, x64]
```

Comment:
I've look at all the build output and certify that these builds area actually doing what they claim. amazing

Reply:
don't forget to mark these required when you merge